scss/components/_icons.scss

```diff
@@ -113,3 +113,18 @@ h6 {
     }
   }
 }
+@media screen and (-ms-high-contrast: active) {
```

Comment:
@geppy
I wasn't sure in which file to add this code so adding it here, is that ok?

Reply:
The normal  definition for `.rounded-circle` in bootstrap package